packages/block-editor/src/components/link-control/index.js

```diff
@@ -115,6 +115,7 @@ function LinkControl( {
 	suggestionsQuery = {},
 	noURLSuggestion = false,
 	createSuggestionButtonText,
+	richPreviews = false,
```

Comment:
I still like `hasRichPreviews` better 😅, any reason why you went for just `richPreviews`?

Reply:
To me the word "has" implies something is "currently" X or Y. When you write a component or pass a prop you are thinking about what it _will_ be in the future once it has rendered.

Therefore I was thinking for simplicity just saying:

> `richPreviews` are they "on" or "off"?

This is not super important to me however, so I'm happy to change if you feel it's better as "has".

Reply:
I changed it 👍

Reply:
I always use `is`, `has`, `should`, or `will` prefix to indicate the variable is a boolean. That's just my personal preference though!

> When you write a component or pass a prop you are thinking about what it will be in the future once it has rendered.

That's an interesting interpretation! I don't think it that way though. Normally I would think of a component as a function for UI, the arguments are props, and the return value is the UI state of those props. If we put `state` aside, then it's just a pure function which maps props to a UI, so I won't say that it's "will", but "has".

But again, it's just a preference thing, I respect your decision 👍.